foundation-model-stack/fms-model-optimizer
# ci: Replace coverage with pytest-cov plugin for code coverage

### Description of the change

[pytest-cov](https://github.com/pytest-dev/pytest-cov) is a coverage plugin for [pytest](https://github.com/pytest-dev/pytest) and provides extra capability on top of [coverage](https://github.com/nedbat/coveragepy) `run`.

## How to verify the PR

One of 2 ways:
- Run locally: `tox -e unitcov`
- Check CI gate `test` workflow that it generates code coverage statistics in the output

### Was the PR tested
